software/multi-scrobbler.yml

```
@@ -0,0 +1,13 @@
+name: multi-scrobbler
+website_url: https://foxxmd.github.io/multi-scrobbler
+source_code_url: https://github.com/FoxxMD/multi-scrobbler
+description: Scrobble music from multiple sources to multiple clients.
+licenses:
+  - MIT
+platforms:
+  - Nodejs
+  - Javascript
+  - Docker
```

```suggestion
platforms:
  - Nodejs
  - Docker
```

> https://awesome-selfhosted.net/platforms/javascript.html
> Projects listed here only require serving static HTML/JS files to clients.

Only Nodejs is relevant here